DemoApp/ios/Podfile.lock

```diff
@@ -319,11 +319,11 @@ PODS:
     - React-cxxreact (= 0.64.1)
     - React-jsi (= 0.64.1)
     - React-perflogger (= 0.64.1)
-  - RNSnowplowTracker (1.0.0):
+  - RNSnowplowTracker (0.2.0):
```

Comment:
Will we bump this to 1.0.0 for release?

Reply:
Yes, i was thinking to leave the change for the prepare-for-release commit, so that all the version changes appear together. The version would need to be changed in:
 - package.json
 - the relevant android-bridge file
 - the relevant ios-bridge file
 - Podfile.lock in DemoApp/ios

Reply:
Sounds good to me!